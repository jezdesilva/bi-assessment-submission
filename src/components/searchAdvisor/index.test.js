import React from 'react';
import { shallow } from 'enzyme';
import SearchAdvisor from './index';

describe('SearchAdvisor', () => {
  const QUERY = 'test';
  const wrapper = shallow(<SearchAdvisor query={QUERY} />);

  test('it renders without crashing', () => {
    shallow(<SearchAdvisor query="test" />);
  });

  test('should match the snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });

  test('renders the message correctly', () => {
    expect(wrapper.find('mark').text()).toEqual(QUERY);
  });
});