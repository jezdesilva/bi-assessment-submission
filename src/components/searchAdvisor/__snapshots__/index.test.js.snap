// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchAdvisor should match the snapshot 1`] = `"<div>Search results for &quot;<mark class=\\"queryHighlight\\">test</mark>&quot;</div>"`;
